src/persist-client/src/cfg.rs

```diff
@@ -416,6 +419,27 @@ pub const TXN_USE_CRITICAL_SINCE: Config<bool> = Config::new(
     "Use the critical since (instead of the overall since) when initializing a subscribe.",
 );
 
+/// Migrate the txns code to use the critical since when opening a new read handle.
+pub const CATALOG_USE_CRITICAL_SINCE: Config<bool> = Config::new(
+    "persist_catalog_use_critical_since",
+    false,
+    "Use the critical since (instead of the overall since) for the Persist-backed catalog.",
+);
+
+/// Migrate the txns code to use the critical since when opening a new read handle.
+pub const SOURCE_USE_CRITICAL_SINCE: Config<bool> = Config::new(
+    "persist_source_use_critical_since",
+    true,
+    "Use the critical since (instead of the overall since) in the Persist source.",
+);
+
+/// Migrate the txns code to use the critical since when opening a new read handle.
+pub const SNAPSHOT_USE_CRITICAL_SINCE: Config<bool> = Config::new(
+    "persist_snapshot_use_critical_since",
```

Comment:
I'd be inclined to call these all `persist_use_critical_since_<suffix>` so they sort together.

Reply:
Including the pre-existing `txn` config? (I named the new ones to be consistent with that older flag, though I guess since we haven'f configured that flag for anyone yet it's probably okay to rename?)

Reply:
Yep! Indeed looks like that one isn't in LD, so super cheap to rename it.